backend/api/views.py

```
@@ -1,3 +1,96 @@
-from django.shortcuts import render
+# api/views.py
+from rest_framework import viewsets
+from rest_framework.decorators import action
+from rest_framework.response import Response
```

Ruff (0.8.2)

`rest_framework.response.Response` imported but unused

Remove unused import: `rest_framework.response.Response`

(F401)